### ISSUE TITLE
Does it support file upload?

### ISSUE THREAD

**Issue by PierrunoYT** [lobehub/lobe-chat#1519]
Does it support file upload?

**Comment by arvinxx** [comment]
by no. but we will make it.

**Comment by twilwa** [comment]
seconding this! the one feature this is missing for me to cancel my other subscriptions lol

**Comment by arvinxx** [comment]
@twilwa We will make it in the next month~

**Comment by dctmfoo** [comment]
This one feature would make this tool so much better. thank you!

**Comment by rickdgit** [comment]
> @twilwa We will make it in the next month~

Hey Just want to follow up on this feature as this is almost the end of the month.
Thanks!

**Comment by arvinxx** [comment]
@rickdgit Plan can't keep up with the changes, we will start to make it in next month 😅

**Comment by blorby** [comment]
file upload would be great, it's still in beta on chatgpt UI but some other modules have it publicly available.
I don't think it's a critical feature but very nice to have.

**Comment by AxDSan** [comment]
> @twilwa We will make it in the next month~



> @rickdgit Plan can't keep up with the changes, we will start to make it in the next month 😅

Let me guess... next month? 😏

**Comment by arvinxx** [comment]
> > @twilwa We will make it in the next month~
> 
> 
> 
> 
> 
> 
> 
> > @rickdgit Plan can't keep up with the changes, we will start to make it in the next month 😅
> 
> 
> 
> Let me guess... next month? 😏

This month after 1.0 release😅__

**Comment by AxDSan** [comment]
[Version 1.1.14](https://github.com/lobehub/lobe-chat/compare/v1.1.13...v1.1.14) still looking for this really desperately! are you guys holding on because of what? is there a proper discussion for this? are you guys waiting on for what? we going to implement RAG? so many questions :)

**Comment by arvinxx** [comment]
@AxDSan We have lots of work to do, but there is only me to implement RAG. Please understand.

**Comment by AxDSan** [comment]
> @AxDSan We have lots of work to do, but there is only me to implement RAG. Please understand.

Totally understandable, if there's anything I could cooperate I would try to do so

**Comment by sjbr** [comment]
We could expect this feature next month, I guess.

**Comment by danisvaliev001** [comment]
Has it been released in [v1.6.14](https://github.com/lobehub/lobe-chat/releases/tag/v1.6.14) and [v1.6.15](https://github.com/lobehub/lobe-chat/releases/tag/v1.6.15)?

**Comment by arvinxx** [comment]
@danisvaliev001 not yet. Please wait for about 1~2 weeks. You can track it here: #3023
